<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainwindowClass</class>
 <widget class="QWidget" name="MainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mainwindow</string>
  </property>
  <widget class="QLineEdit" name="port_edit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openServer_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开启服务器</string>
   </property>
  </widget>
  <widget class="QTableView" name="queue_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>481</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="info_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>481</width>
     <height>161</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="msg_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>481</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearMsg_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空消息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>580</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
